### PR TITLE
docs minor corrections [skip ci]

### PR DESCRIPTION
**Pull request by vchenin**

### CODE REVIEW

File: docs/guide/topics-creating-your-own-templates.md [hunk 1]
```diff
@@ -28,9 +28,6 @@ Now open this folder and modify any template to fit your desires, for example, a
 
     <?= "<?php " ?>$form = ActiveForm::begin(); ?>
     <?= "<?=" ?> $form->errorSummary($model) ?> <!-- ADDED HERE -->
-    <?php foreach ($safeAttributes as $attribute) {
```
Comment (vchenin) [inline docs/guide/topics-creating-your-own-templates.md]: obsolete code

File: docs/guide/topics-creating-your-own-templates.md [hunk 2]
```diff
@@ -28,9 +28,6 @@ Now open this folder and modify any template to fit your desires, for example, a
...
-        echo "    <?= " . $generator->generateActiveField($attribute) . " ?>\n\n";
-    } ?>
```
Comment (vchenin) [inline docs/guide/topics-creating-your-own-templates.md]: obsolete code

Comment (samdark) [inline docs/guide/topics-creating-your-own-templates.md]: It should be updated then, not removed.

Comment (vchenin) [inline docs/guide/topics-creating-your-own-templates.md]: @samdark this code is not significant in present case

Comment (samdark) [inline docs/guide/topics-creating-your-own-templates.md]: How would it generate form fields then?

Comment (vchenin) [inline docs/guide/topics-creating-your-own-templates.md]: @samdark in this example just part of code is shown, so it's not necessary how many lines of code are presented.
We have reference-point:
```php
    <?= "<?php " ?>$form = ActiveForm::begin(); ?>
```
and line which was inserted:
```php
    <?= "<?=" ?> $form->errorSummary($model) ?> <!-- ADDED HERE -->
```
the rest are not significant in this case:
```
//...
```

Comment (samdark) [inline docs/guide/topics-creating-your-own-templates.md]: Ah. Right. OK.

**Comment by samdark** [comment]
Thanks!

**Pull request merged**